src/components/pages/state/charts/index.js

```diff
@@ -41,7 +44,15 @@ const getDataForField = (data, field) => {
 // we need a combined bar + line chart
 
 export default ({ history }) => {
-  const data = [...history].sort((a, b) => a.date - b.date)
+  // Change below to update range of chart
+  const NUM_DAYS = 90
```

Comment:
In the future, let's move these kind of variables to `gatsby-config.js`'s site metadata

Reply:
sounds good, will do.